source/CMakeLists.txt

```diff
@@ -90,22 +90,22 @@ add_library(reminecraftpe-core STATIC
     client/player/input/MouseTurnInput.cpp
     client/player/input/KeyboardInput.cpp
     client/player/input/ITurnInput.cpp
-	client/player/input/IBuildInput.cpp
-	client/player/input/IncludeExcludeArea.cpp
-	client/player/input/MouseHandler.cpp
-	client/player/input/PolygonArea.cpp
-	client/player/input/RectangleArea.cpp
-	client/player/input/MouseDevice.cpp
-	client/player/input/Multitouch.cpp
-	client/player/input/CustomInputHolder.cpp
-	client/player/input/IInputHolder.cpp
-	client/player/input/IMoveInput.cpp
-	client/player/input/ITouchScreenModel.cpp
-	client/player/input/TouchAreaModel.cpp
-	client/player/input/TouchInputHolder.cpp
-	client/player/input/TouchscreenInput_TestFps.cpp
-	client/player/input/UnifiedTurnBuild.cpp
     client/network/ClientSideNetworkHandler.cpp
+    client/player/input/IBuildInput.cpp
```

Comment:
Why were these moved to insert `ClientSideNetworkHandler.cpp` in the middle of the `client/player/input` group?

Reply:
Because I started this PR beefire you fixed tge CMake and the rebase messed things up. I'll fix it.

Reply:
This is fixed now.